src/EditorFeatures/Core/Extensibility/NavigationBar/NavigationBarItems/RoslynNavigationBarItem.GenerateDefaultConstructorItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Editor.Extensibility.NavigationBar
+{
+    internal abstract partial class RoslynNavigationBarItem
+    {
+        public class GenerateDefaultConstructor : AbstractGenerateCodeItem
+        {
+            public GenerateDefaultConstructor(string text, SymbolKey destinationTypeSymbolKey)
+                : base(RoslynNavigationBarItemKind.GenerateDefaultConstructor, text, Glyph.MethodPublic, destinationTypeSymbolKey)
```

Comment:
Are the kinds for a follow-up where we're actually serializing this?

Reply:
yes.